roles/ceph-common/tasks/main.yml

```diff
@@ -29,7 +29,7 @@
 
 - name: install ceph
   apt: pkg={{ item }}
-       state=present
+       state=latest
```

Comment:
I really don't like using `latest` here. That really feels like a recipe for disaster if we accidentally publish a newer package to the repository.